go/kubebuilder/memcached-operator/controllers/memcached_controller.go

```diff
@@ -1,4 +1,6 @@
 /*
+
+
```

Comment:
This is surprising

Reply:
space here is how it is gen. When the project/sample was created it was manually changed